backend/src/main/java/gov/cdc/usds/simplereport/api/pxp/PatientSelfRegistrationController.java

```diff
@@ -112,7 +113,9 @@ public boolean isExistingPatient(
 
   @GetMapping("/entity-name")
   public String getEntityName(
-      @RequestParam String patientRegistrationLink, HttpServletRequest request) {
+      @RequestParam @Pattern(regexp = "[2346789abcdefghjkmnpqrtuvwxyz]{5}")
```

Comment:
I wanted to build this pattern string from the constants defined in `PatientSelfRegistrationLink.java` but, alas, `The value for annotation attribute Pattern.regexp must be a constant expression`. This will be hardcoded unless someone knows of a workaround.